src/main/flight/pid.c

```diff
@@ -248,7 +248,7 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
         // -----calculate P component and add Dynamic Part based on stick input
         float PTerm = Kp[axis] * errorRate * tpaFactor;
         if (axis == FD_YAW) {
-            PTerm = ptermYawFilterApplyFn(ptermYawFilter, PTerm);
+            PTerm = yawNotchFilterApplyFn(yawFilterNotch, PTerm);
```

Comment:
As per other comment, this should retain the name `ptermYawFilterApplyFn` - that is the logical name (as opposed to physical name) of the filter.